backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/CoreFeaturesSwitch.java

```diff
@@ -93,6 +103,15 @@ public List<EStructuralFeature> caseFeature(Feature object) {
         return features;
     }
 
+    @Override
```

Comment:
Please put it at the appropriate location (all cases are ordered alphabetically)